src/commands/cmd_key.cc

```diff
@@ -236,12 +236,16 @@ class CommandPersist : public Commander {
 class CommandDel : public Commander {
  public:
   Status Execute(Server *svr, Connection *conn, std::string *output) override {
-    int cnt = 0;
-    redis::Database redis(svr->storage, conn->GetNamespace());
+    std::vector<rocksdb::Slice> keys;
```

Comment:
Could we call `reserve` here for the vector before adding the known number of elements to it?

Reply:
Others LGTM.